privacyidea/static/components/config/views/config.events.details.html

```diff
@@ -102,6 +102,10 @@
             <span class="glyphicon glyphicon-plus"></span>
             <span translate>Save Event Handler Definition</span>
         </button>
+        <a target="_external" href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+        privacyideaVersionNumber+'/eventhandler/index.html#'}}" class="btn btn-primary">
+            <span class="glyphicon glyphicon-book"></span> doc.
+        </a>
```

Comment:
Currently the help button is only visible if the admin has the right `eventhandling_write`.
A helpdesk user might also be interested in understanding the configuration of the event handlers, even if he is not allows to write the config. So we should make the help always visible.

![image](https://user-images.githubusercontent.com/1908620/171847987-05020b05-3738-402a-a492-8ce0a8714557.png)

since it is a link to the top view of the help, it might be a good idea to add the help link on the left side underneath

"Alle Ereignisse"
"Erstelle ein neues Ereignies"

This way we could always add help links on the left side, for users, for tokens, for configs...

Reply:
I personally would remove the buttons and suggest one help link in the left menu

* Alle Ereignisse
* Erstelle Neues Ereignies
* Hilfe zu Ereignissen

Reply:
Maybe like this 95186c7e1b49a1f3caaee95daa5b5361b6f340a0?